x-pack/qa/multi-cluster-search-security/legacy-with-basic-license/build.gradle

```diff
@@ -0,0 +1,76 @@
+import org.elasticsearch.gradle.internal.info.BuildParams
+import org.elasticsearch.gradle.internal.test.RestIntegTestTask
+
+apply plugin: 'elasticsearch.standalone-rest-test'
+apply plugin: 'elasticsearch.rest-resources'
+
+restResources {
+  restApi {
+    include '_common', 'bulk', 'field_caps', 'security', 'search', 'clear_scroll', 'scroll', 'async_search', 'cluster',
+      'indices', 'open_point_in_time', 'close_point_in_time', 'terms_enum'
+  }
+}
+
+// randomise between sniff and proxy modes
+boolean proxyMode = (new Random(Long.parseUnsignedLong(BuildParams.testSeed.tokenize(':').get(0), 16))).nextBoolean()
+
+def fulfillingCluster = testClusters.register('fulfilling-cluster') {
+  setting 'xpack.security.enabled', 'true'
+  setting 'xpack.license.self_generated.type', 'basic'
+  module ':modules:analysis-common'
+  module ':modules:lang-painless'
+  module ':modules:data-streams'
+  module ':x-pack:plugin:mapper-constant-keyword'
+  module ':x-pack:plugin:async-search'
```

Comment:
since we are not using the default distribution, we need to specify each of the modules that are in use at runtime.